### PR TITLE
Persian (Farsi) translation

### PR DESCRIPTION
**Pull request by yousefvand**
## Description

Persian translation

## Affected issues

None

## Pull request checklist

* [ ] I have tested my code
* [x] I have read the [contributing guide](https://github.com/AnarchyLinux/installer/blob/HEAD/CONTRIBUTING.md)
* [x] I have followed best practices and commented my code well

**Note:** Persian (Farsi) is a RTL language and may not be displayed correctly in terminal (AFAIK it is OK out of the box in `konsole` while it breaks in `GNOME terminal` and needs some hacks to work).

### CODE REVIEW

File: extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dash-to-panel@jderose9.github.com/README.md [hunk 1]
```diff
@@ -103,4 +103,4 @@ Thanks to the following people for contributing via pull requests:
 - @MartinPL for toggling favorites on/off in panel
 - @jackwickham for thumbnail middle and right click actions
 - Bug Fixes: @imrvelj, @Teslator, @bil-elmoussaoui, @brandon-schumann, @sw9, @rockon999
-- Translations: @frnogueira (pt_BR), @zeten30 (cs_CZ), @franglais125 (es), @LaurentTreguier / @SolarLiner (fr), @elsieholmes (uk), @hosiet (zh\_CN), @jonnius (de), @urbalazs (hu), @crayxt (kk), @pkomur (pl), @AlexGluck (ru)
\ No newline at end of file
+- Translations: @frnogueira (pt_BR), @zeten30 (cs_CZ), @franglais125 (es), @LaurentTreguier / @SolarLiner (fr), @elsieholmes (uk), @hosiet (zh\_CN), @jonnius (de), @urbalazs (hu), @crayxt (kk), @pkomur (pl), @AlexGluck (ru), @yousefvand (fa_IR)
```
Comment (erazemkokot) [inline extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dash-to-panel@jderose9.github.com/README.md]: These translators are for the dash-to-panel GNOME extension afaik, not for Anarchy, so if you didn't translate that, I wouldn't add myself to there.

File: extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui [hunk 2]
```diff
@@ -19,6 +19,7 @@ Mosaab Alzoubi (moceap)
 Alonso Lara (AlonsoLP)
 Dingzhong Chen (wsxy162)
 narzb
+Remisa Yousefvand (yousefvand)
```
Comment (erazemkokot) [inline extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui]: Same here

Comment (yousefvand) [reply extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui]: I was not sure about this one. By removing the other one, there would be nowhere to add translator, while instructions told add yourself. I'd leave it up to your team.
BWT I'd like to work on using docker as ISO building incubator (without `Archiso` at this step). Is it OK?

Comment (erazemkokot) [reply extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui]: You did add yourself in the anarchy-persian.conf. Sorry if the instructions weren't clear enough.
By add yourself I meant add your name and email in the translation file, which you did properly.

Comment (erazemkokot) [reply extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui]: > BWT I'd like to work on using docker as ISO building incubator (without `Archiso` at this step). Is it OK?

Of course. I'd prefer if the Dockerfile is in the root directory of the repo, so it's easier for new users to find, it that seems reasonable to you.

Comment (yousefvand) [reply extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui]: Sure I'd make it in project root and update you with result.

Comment (erazemkokot) [reply extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui]: Thanks

**Comment by yousefvand** [comment]
On a VM (debian) after cloning the project to home I had it at `~/installer`. Since `makepkg` doesn't work as root [this](https://github.com/greyltc/docker-archlinux-aur/blob/master/add-aur.sh) script should be placed at  `~/installer` too. Then I started a container in an interactive mode to test if an ISO can be generated successfully:
 `$ docker run --name=test -v ~/installer:/anarchy -it archlinux/base`
inside container as root (create user `anarchy`):
```
# cd /anarchy
# bash add-aur.sh anarchy
```
before running `compile.sh` I did change  `makepkg` commands (two instances are there) to:
`su anarchy -c "yay -S --needed --noprogressbar --needed --noconfirm ${pkg}"`
Then from inside interactive container shell:
`# bash compile.sh`
Everything was good until a `sudo: setrlimit` error stopped the process:
z
This [bug](https://bugs.archlinux.org/task/65713) has been reported 3 days ago.
I'm not proud of this approach, even after bug fix, a docker image based on this approach would be too large. I'll try `Archiso` approach later.

**Comment by yousefvand** [comment]
Sadly it seems there is no easy way to use RTL in terminal, so this PR could be closed.

**Pull request closed without merging**